### ISSUE TITLE
Create Roadmap page & section on homepage

### ISSUE THREAD

**Issue by tomaroberts** [GSTT-CSC/csc-test-site#21]
Issue to track the creation of the Roadmap page.

Currently, this is only located on the front page. I suggest we have a dedicated Roadmap page (under About Us or Work?) which has the full Roadmap. Then, we can have an abridged version on the front page, which has the major points.

Within the site structure, we can have a `_data/roadmap.yml` which contains ALL the Roadmap content, then cherry-pick from that content for the front page section. So there would be:
- `_includes/roadmap.html` for the main Roadmap page
- `_includes/roadmap-abridged.html` for the front page section

**Comment by dangerdika** [comment]
@lucy-funnell and will finalise the content and i'll get onto implementing it ASAP

**Comment by lucy-funnell** [comment]
@dangerdika I have emailed you the template I started, feel free to make edits/changes or start from scratch
Let me know if I can provide any other details, more than happy to help build on this timeline further.

**Comment by tomaroberts** [comment]
@dangerdika @lucy-funnell 

Not sure where you guys got to with this, but I made a new branch called `roadmap` where I started fiddling with this. The existing code for the Roadmap section is pretty complicated from a quick look. The .scss and Liquid code are quite involved. It's actually pretty cool how it's implemented. ANYWAY, I think we had in mind doing the Roadmap horizontally, but given the complexity of the code it might be easiest to just leave the vertical format as is? Especially whilst we just want to get the website up and running.

At the moment it looks like this:

<img width="1386" alt="image" src="https://user-images.githubusercontent.com/15610752/167681663-4dbc191c-51cd-461e-a3f8-44f441357d22.png">

It needs:
- Pictures (200x200px) 
- Blurb for each point on the Roadmap. @lucy-funnell – sounds like you started writing this?

The front page of the site is nearing completion (v0.1.0... at least...), so would be cool to get the Roadmap part sorted too.
